src/constants.js

```diff
@@ -20,10 +20,8 @@ import {
   _SENTRY_DSN as SENTRY_DSN,
 } from './config';
 
-hathorLib.storage.setStore(new AsyncStorageStore());
-hathorLib.storage.setItem('wallet:server', 'https://mobile.wallet.hathor.network/v1a/');
-hathorLib.network.setNetwork('mainnet');
-
+export const STORE = new AsyncStorageStore()
+hathorLib.storage.setStore(STORE);
```

Comment:
Shouldn't we pass the store to the wallet instead of directly calling `hathorLib.storage.setStore()`?

Reply:
This is one of the parts of the code that I left to a later refactor. The storage is needed in the whole wallet, not only when we have a wallet object. Because of that we need to have a global store in the lib (we have some keys that are used in the wallet, e.g. 'loaded'). I believe removing this store dependency should be in a different refactor.

Reply:
I got it. Let's just add a comment here explaining why it is needed and and it should be refactored later on.

Reply:
Done